### PR TITLE
Update duct version (connect #1224)

### PR DESCRIPTION
**Pull request by tangrammer**
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: backend/dev/src/dev.clj [hunk 1]
```diff
@@ -2,32 +2,53 @@
...
-  (load-system (keep io/resource
-                     ["akvo/lumen/system.edn" "dev.edn" "local.edn"])))
+(defn read-config []
+  (duct/read-config (io/resource "dev.edn")))
+
+(derive :akvo.lumen.component.emailer/dev-emailer :akvo.lumen.component.emailer/emailer)
+(derive :akvo.lumen.component.caddisfly/local :akvo.lumen.component.caddisfly/caddisfly)
+(derive :akvo.lumen.component.error-tracker/local :akvo.lumen.component.error-tracker/error-tracker)
+
+(defn dissoc-prod-components [c]
```
Comment (iperdomo) [inline backend/dev/src/dev.clj]: Not sure why are directly referencing the production one and then removing it at dev time, instead we should reference the generic *parent* one?

https://github.com/weavejester/integrant#derived-keywords

Comment (tangrammer) [reply backend/dev/src/dev.clj]: After talking with @iperdomo we agree on ignoring this comment 
(Expanded bits are that integrant needs the implementation (child) keyword in dev.edn or config.edn thus integrant will search for the init-key defmethod for the generic (parent) keyword component to be initiated)

**Comment by tangrammer** [comment]
@kardan aligned with `develop`

**Comment by tangrammer** [comment]
@kardan rebased with develop including export service now :)

**Comment by kardan** [comment]
I could not get local configs (from local.edn) to work properly. I looked at the default Duct template and it looked like an empty local.clj file was created so did that locally but that didn't pick up local configs. Need to sign out for the day, will continue to look at it tomorrow but @tangrammer did you try this?

**Comment by tangrammer** [comment]
@kardan I didn't commit local.edn nor added to settings thus i didn't know how not to commit the file to the git repo so i didn't want to track its changes. The problem is more related to how put this empty file in the repo but doesn't track it in git 🤔 ... maybe you know better how to do it or if it's worthing 


anyway there is no problem to add it and get it working

**Pull request merged**